src/main/java/org/sonarlint/intellij/ui/ruledescription/RuleLanguages.kt

```diff
@@ -56,7 +56,8 @@ enum class RuleLanguages(private val language: Language, private vararg val file
     TERRAFORM(Language.TERRAFORM, "Terraform"),
     TS(Language.TS, *typescriptFileTypesByPriority),
     XML(Language.XML, "XML"),
-    YAML(Language.YAML, "YAML");
+    YAML(Language.YAML, "YAML"),
+    PLSQL(Language.PLSQL, "PL/SQL");
```

Comment:
I don't have `PL/SQL` as a file type in IntelliJ Ultimate, is this from a plugin?